client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -33,6 +41,13 @@ export default function Dashboard(): JSX.Element {
       <Grid item className={classes.drawerWrapper}>
         <ChatSideBanner loggedInUser={loggedInUser} />
       </Grid>
+      <Grid>
+        <Button onClick={openAddProductModal}>Add Product</Button>
+        <AddProduct
```

Comment:
same comment here on no children components can end it off with />

Reply:
I will be more careful next time.. thank you